app/api/routes.py

```diff
@@ -8,17 +8,26 @@
 from app.utils import Paginator, standardize_response, setup_logger
 from dateutil import parser
 from datetime import datetime
+from prometheus_client import Counter, Summary
 import uuid
 
+# Metrics
+failures_counter = Counter('my_failures', 'Number of exceptions raised')
+latency_summary = Summary('request_latency_seconds', 'Length of request')
+
 logger = setup_logger('routes_logger', 'log/routes.log')
 
 
 # Routes
+@latency_summary.time()
```

Comment:
Lots of DRY: should probably use the `@bp.before_request` decorator (in a future PR)

Reply:
I didn't realize there was a `before_request` decorator, that's super useful. Yeah, we'll need to do that